### ISSUE TITLE
Checkinstall on Ubuntu 18.04 LTS throws: make: *** [cmake_check_build_system] Segmentation fault

### ISSUE THREAD

**Issue by sebma** [giuliomoro/checkinstall#5]
Hi,

Checkinstall on Ubuntu 18.04 LTS throws: make: *** [cmake_check_build_system] Segmentation fault : 
```bash
$ sudo checkinstall --pkgname atomicparsley --pkgversion 20210715.151551.e7ad03a --pkgsource=https://github.com/wez/atomicparsley

checkinstall 1.6.2, Copyright 2009 Felipe Eduardo Sanchez Diaz Duran
           This software is released under the GNU GPL.


The package documentation directory ./doc-pak does not exist. 
Should I create a default set of package docs?  [y]: 

Preparing package documentation...OK

*** No known documentation files were found. The new package 
*** won't include a documentation directory.

*****************************************
**** Debian package creation selected ***
*****************************************

This package will be built according to these values: 

0 -  Maintainer: [ root@sebastien-B206 ]
1 -  Summary: [ AtomicParsley is a lightweight command line program for reading, parsing and setting metadata into MPEG-4 files, in particular, iTunes-style metadata. ]
2 -  Name:    [ atomicparsley ]
3 -  Version: [ 20210715.151551.e7ad03a ]
4 -  Release: [ 1 ]
5 -  License: [ GPL ]
6 -  Group:   [ checkinstall ]
7 -  Architecture: [ i386 ]
8 -  Source location: [ https://github.com/wez/atomicparsley ]
9 -  Alternate source location: [  ]
10 - Requires: [  ]
11 - Provides: [ atomicparsley ]
12 - Conflicts: [  ]
13 - Replaces: [  ]

Enter a number to change any of them or press ENTER to continue: 

Installing with make install...

========================= Installation results ===========================
Makefile:556: recipe for target 'cmake_check_build_system' failed
make: *** [cmake_check_build_system] Segmentation fault (core dumped)

****  Installation failed. Aborting package creation.

Cleaning up...OK

Bye.

$ echo $?
1
```

**Comment by sebma** [comment]
@giuliomoro  Thanks a lot. You fixed it in commit 57ad147.

**Comment by sebma** [comment]
This is related to https://github.com/opencv/opencv/issues/8897.
